### ISSUE TITLE
ArcGIS QuickCapture is graduating.

### ISSUE THREAD

**Issue by IsmaelInRedlands** [Esri/QuickCapture#80]
QuickCapture Esri Labs has graduated into a fully supported ArcGIS product. Find the latest info in our GeoNet Place at: https://community.esri.com/community/arcgis-quickcapture

If you are using ArcGIS QuickCapture Esri Labs, you will find detailed information in the [Early Adopter Site](https://earlyadopter.esri.com/key/ArcGISQuickCapture) about how to migrate your existing projects from ArcGIS QuickCapture Esri Labs to ArcGIS QuickCapture.

**Comment by slibby** [comment]
@IsmaelInRedlands  That means that this source code is no longer the latest code for QuickCapture, correct?

**Comment by IsmaelInRedlands** [comment]
That's correct. We will update the source code of QuickCapture when the next release of AppStudio is  made available.
